src/styles/_catalog.scss

```
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+  justify-content: center;
+  align-content: center;
+  margin: 50px 0;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
-    height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    height: 300px;
+  }
+
+  @media (max-width: 1023px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
```

```suggestion
      grid-template-columns: repeat(3, 200px);
```
Fix it everywhere and move 200px to the variable